AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -399,7 +403,7 @@ - (void)_reloadDataWithAnimationOptions:(ASDataControllerAnimationOptions)animat
   _initialReloadDataHasBeenCalled = YES;
   [self performEditCommandWithBlock:^{
     ASDisplayNodeAssertMainThread();
-    [_editingTransactionQueue waitUntilAllOperationsAreFinished];
+    dispatch_sync(_editingTransactionQueue, ^{});
```

Comment:
More descriptive way to do this so it's clear we're waiting for the queue to empty before proceeding? Macro?